pkg/config/config.go

```diff
@@ -69,48 +72,78 @@ type ContainersConfig struct {
 	// and "systemd".
 	CgroupManager string `toml:"cgroup_manager"`
 
+	// Default way to to create a cgroup namespace for the container
+	CgroupNS string `toml:"cgroupns"`
+
 	// Capabilities to add to all containers.
 	DefaultCapabilities []string `toml:"default_capabilities"`
 
-	// DefaultMountsFile is the path to the default mounts file for testing
-	// purposes only.
-	DefaultMountsFile string `toml:"-"`
-
 	// Sysctls to add to all containers.
 	DefaultSysctls []string `toml:"default_sysctls"`
 
 	// DefaultUlimits specifies the default ulimits to apply to containers
 	DefaultUlimits []string `toml:"default_ulimits"`
 
+	// DNS Set default DNS servers.
+	DNS string `toml:"default_ulimits"`
+
+	// DNSOptions Set  default DNS options.
```

Comment:
extra space after set.

Should the various `"Set"` used in the comments be lower case?

Reply:
Fixed